### PR TITLE
Removed comments

### PR DESCRIPTION
**Pull request by vishwasnm1**

### CODE REVIEW

File: Library/src/ReturnBook.java [hunk 1]
```diff
@@ -22,10 +22,7 @@
 	private JPanel contentPane;
 	private JTextField textField;
 	private JTextField textField_1;
-
-	/**
-	 * Launch the application.
-	 */
```
Comment (ghost) [inline Library/src/ReturnBook.java]: Comments are required

**Review by ghost — CHANGES_REQUESTED** [review]
Modify the changes mentioned in the comment

**Review by vishwasnm1 — COMMENTED** [review]
I have made the changes @VishwasNM . 
Review the changes and approve it.

**Review by ghost — APPROVED** [review]
The Changes have been reviewed.

**Pull request merged**